src/Symfony/Component/Intl/Countries.php

```diff
@@ -58,13 +100,46 @@ public static function getName(string $country, string $displayLocale = null): s
     }
 
     /**
+     * Get country name from alpha3 code.
+     *
+     * @throws MissingResourceException if the country code does not exists
+     */
+    public static function getAlpha3Name(string $country, string $displayLocale = null): string
+    {
+        $alpha2Code = self::readEntry(['Alpha2ToAlpha3', $country], 'meta');
```

Comment:
given we pass an alpha3 code, we need a lookup by value to get us the corresponding key (which is the alpha2 code). This wont work IIUC.

Reply:
Yes, I have changed that.